inspire_query_parser/utils/visitor_utils.py

```diff
@@ -42,21 +42,44 @@
                                          DATE_YESTERDAY_REGEX_PATTERN)
 
 
+NAME_INITIAL_FOLLOWED_BY_FIRSTNAME_WITHOUT_SPACE = re.compile(r"(\.[a-z])", re.IGNORECASE)
+
+
+def author_names_strip_empty_and_maybe_split_if_initial_followed_by_first_name_without_space(names):
```

Comment:
```suggestion
def retokenize_first_names(names):
```
I think this should be moved to `ParsedName`, but this is OK for a quick-fix.